changes/8892.skymatch.rst

```diff
@@ -0,0 +1 @@
+Resolve warnings emitted by NumPy 2 when running skymatch.
```

Comment:
I wonder what is the policy for when change log entries are needed. This PR does not change API or provide any particular enhancement (to the user) or bug fix.